### ISSUE TITLE
[Question] Meaning of "Best validation score" form sprint_statistics()

### ISSUE THREAD

**Issue by Mont-Mi-Rail** [automl/auto-sklearn#1538]
Hello guys, quick question.

What does "Best validation score" form sprint_statistics() means ? I understand it's the best score of all the validation scores, but how are those validation scores calculated and what do they mean ? 

Thanks a lot !

**Comment by eddiebergman** [comment]
Hi @Mont-Mi-Rail,

Auto-sklearn will train a large different set of models and trains them with a `training` set and then evalutes them on a `validation` set. The "Best validation score" is the best score achieved on this `validation set`.

Best,
Eddie

**Comment by Mont-Mi-Rail** [comment]
Ok thanks a lot !

And this validation score is like an accuracy score I suppose ?

**Comment by eddiebergman** [comment]
Depends on the metric you set or the default in case you didn't set one.

You can find the default metrics here!
https://github.com/automl/auto-sklearn/blob/12fe4495f905ce02424094f1006cdf8df809f351/autosklearn/metrics/__init__.py#L753-L759

**Comment by Mont-Mi-Rail** [comment]
Perfect ! Thanks a lot !
